### DISCUSSION TITLE
Payments using Debit Card from India

### DISCUSSION THREAD

**Discussion by rahulkatiyar19955** [community/community#30737] [Copilot Conversations]
I was unable to pay the subscription fees for Github Copilot using the Debit Card. Does anyone has any solution?

**Comment by iXingo** [comment]
you can use paypal

**Reply by Wep-wep** [reply]
Paypal

**Reply by rahulkatiyar19955** [reply]
On clicking `Pay Now` button on billing section, it is asking for only credit card while the paypal is already configured
<img width="972" alt="screenshot" src="https://user-images.githubusercontent.com/24501644/186944518-33ab4b9e-dbce-4ca6-a8c5-0141a9100d08.png">
.

**Comment by iamcjwilliams** [comment]
Are you still experiencing this issue? @rahulkatiyar19955

**Reply by rahulkatiyar19955** [reply]
Yes, still facing the issue.

**Reply by kolumnin** [reply]
Have you tried Paytm?

**Comment by iamcjwilliams** [comment]
Hello @rahulkatiyar19955 Please use this[ link](https://docs.github.com/en/billing/managing-your-github-billing-settings/about-billing-on-github), if you're still facing this issue, can you provide more detail once you've gone through the steps

**Comment by OmkarK45** [comment]
Hey did you figure out any solution? 
@rahulkatiyar19955

**Reply by rahulkatiyar19955** [reply]
Looks like the problem is solved last month, sorry for not notifying here. 
Related Info: you need to activate international transaction on your debit card, also if you fail to enter the correct otp first time, it will take entire day or two just to retry the payment. If you try to proceeds within that time period it simply says transaction fails.

**Reply by OmkarK45** [reply]
Which card were you using ?
Visa, mastercard?
I have rupay debit it says card declined (i have intl tnx enabled)

**Comment by myexsite** [comment]
I am also facing the same issue. I am using PayTM debit card with international transaction enabled. Is there any other way to pay to github in india. I don't have a credit card

**Reply by meta-ks** [reply]
Hi @iamcjwilliams 
I am trying to use paypal to make payment (from India) but getting error: 
![image](https://github.com/community/community/assets/24271600/89ea47fa-30e9-4875-85b0-f5ddbadf16f0)
I tried multiple times but still facing the same issue.

**Comment by oootkarsh** [comment]
Facing the same issue with Kotak card
